tests/yaml/issue-77.yaml

```diff
@@ -16,6 +16,7 @@ table: |
   include tables/latinLetterDef6Dots.uti
   punctuation . 256
   punctuation - 36
+  display \s 6
```

Comment:
How is this related to the rest of the PR?

Reply:
This test was failing because the 6 from the rule below had no display rule. It used to fall back to a space, so now I had to make that explicit.

Reply:
Ah, ok, makes sense